valheim_plus.cfg

```diff
@@ -66,6 +66,18 @@ noWeatherDamage=false
 ; The maximum range that you can place build objects at
 maximumPlacementDistance=5
 
+; When destroying a building piece, setting this to true will ensure it always drops full resources.
```

Comment:
Would you be able to add a line `; default: false` etc for these configurations? Not technically a convention we have listed but more a preference